recipe/meta.yaml

```
@@ -32,24 +32,24 @@ requirements:
     - packaging
     - pip
     - python
-    - setuptools
     - wheel
   run:
     - ipython
     - jinja2 >=2.1
     - jupyter_core
     - jupyter_server >=1.16,<3
-    - jupyterlab_server >=2.10,<3
+    - jupyter_server_ydoc >=0.8.0,<0.9.0
+    - jupyter_ydoc >=0.2.3,<0.3
+    - jupyterlab_server >=2.13,<3
     - nbclassic
     - notebook <7
-    - packaging
     - python
     - tomli
```

see https://github.com/jupyterlab/jupyterlab/blob/3dea0ef213db2b44b5634efc20245e0034744932/setup.cfg#L46
```suggestion
    - tomli  # [py<311]
```